doc/source/dev/roadmap.rst

```diff
@@ -50,13 +50,14 @@ requested in this respect.
 Statistics enhancements
 -----------------------
 
-The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
-particularly high importance to the project.
-
-- Improve the options for fitting a probability distribution to data.
-- Expand the set of hypothesis tests.  In particular, include all the basic
-  variations of analysis of variance.
-- Add confidence intervals for all statistical tests.
```

Comment:
Removing enhancements, as they don't seem quite as important as essential maintenance below. All of these items are still in the detailed roadmap; now there is no more duplication.